source/sync/configure/sync-settings.txt

```diff
@@ -173,11 +201,64 @@ Performance and Storage
 Each queryable field adds additional metadata storage to your Atlas cluster and
 may lead to degraded write performance. You should have as few queryable fields
 as needed by your application. A good rule of thumb is to have at most 10
-queryable fields.
+queryable fields. If you need to reduce storage usage or improve performance, 
+you can remove unneeded queryable fields from your App. However, be aware
+of the consequences of adding or removing queryable fields detailed below.
+
+For additional considerations, refer to :ref:`optimizing performance and 
+storage when using Flexible Sync <optimizing-performance-and-storage-flexible-sync>`.
+
+.. _fs-indexed-queryable-fields:
+
+Indexed Queryable Fields
+++++++++++++++++++++++++
+
+You can improve performance for certain types of workloads by adding an
+indexed queryable field. An **indexed queryable field** is a 
+**global queryable field** that is indexed in the history store, providing
```

Comment:
I think `history store` is maybe leaking a bit implementation detail than is needed here. I think something like this is probably enough information

`An **indexed queryable field** is a special **global queryable field** that can be queried on more efficiently, providing improved Sync performance.`